uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/modules/models.js

```diff
@@ -1,5 +1,6 @@
 define([
   'common/angular',
+  'common/angular-file-upload',  
```

Comment:
I was getting this before `Unknown provider: FileUploaderProvider <- FileUploader <- FileUpload <- LeaveRequestInstance <- LeaveRequest`